src/Creator/RefundUnitsCommandCreator.php

```diff
@@ -46,6 +48,13 @@ public function fromRequest(Request $request): Command
     private function parseIdsToUnitRefunds(array $units): array
     {
         return array_map(function (array $refundUnit): UnitRefund {
+            if (isset($refundUnit['amount']) && $refundUnit['amount'] !== '') {
+                $id = (int) $refundUnit['partial-id'];
```

Comment:
I wonder whether using a form to create a command wouldn't be a better option - right now we have to use custom code for both command creation, form handling and form view rendering which can't be extended or reused as easily as forms (total would be `MoneyType` etc.).